openshift/datasync-http.yml

```diff
@@ -2,22 +2,21 @@
 kind: Template
 apiVersion: v1
 metadata:
-  name: datasync-http
+  name: datasync-server-app
   annotations:
-    openshift.io/display-name: Mobile Developer Services Data Sync Gateway
-    description: Mobile Developer Services DataSync template
+    openshift.io/display-name: Data Sync App
+    description: Template allows to deploy DataSync Server application from source code
     tags: sync, mobile, nodejs
     iconClass: icon-nodejs
-    openshift.io/long-description: This template builds Node.js based applications
-      integration with Data Sync.
+    openshift.io/long-description: Template allows to deploy DataSync Server application from source code
     openshift.io/provider-display-name: Red Hat, Inc.
     openshift.io/documentation-url: https://github.com/aerogear/datasync-deployment
     openshift.io/support-url: https://access.redhat.com
     template.openshift.io/bindable: 'false'
 message: 'The following service(s) have been created in your project: ${NAME}.'
 labels:
-  template: data-sync-http
-  app: data-sync-http
+  template: data-sync-server-app
```

Comment:
In openshift, the name in the UI is appearing as "data-sync-http".
I'm think that it would make more sense to set this value to "Data Sync App".

NOTE: I'm not sure if it's this parameter or the paramter below (or neither of these) that needs to be changed though.

Reply:
I have renamed it here but obviously it is not on master. If we use templates from this branch name will bw changed. Name cannot have spaces so I cannot apply this suggestion.